wombat/src/main/include/utils/Gearbox.h

```diff
@@ -1,32 +1,34 @@
-#pragma once 
+#pragma once
 
-#include "utils/Encoder.h"
-#include "VoltageController.h"
 #include <frc/system/plant/DCMotor.h>
 
+#include "VoltageController.h"
+#include "utils/Encoder.h"
+
 namespace wom {
+namespace utils {
   using DCMotor = frc::DCMotor;
-/**
- * Struct for motor and encoder pairs.
- *
- * Combines the motor and encoder parts into one data structure,
- * so that both Spark + Encoder and Talon SRX are treated the same.
- */
-struct Gearbox {
-
   /**
-   * The VoltageController (Motor Controller). May not be null.
+   * Struct for motor and encoder pairs.
+   *
+   * Combines the motor and encoder parts into one data structure,
+   * so that both Spark + Encoder and Talon SRX are treated the same.
    */
-  VoltageController *transmission;
+  struct Gearbox {
+    /**
+     * The VoltageController (Motor Controller). May not be null.
+     */
+    VoltageController *transmission;
 
-  /**
-   * The Encoder. May be null, depending on the consumer of this structure.
-   */
-  Encoder *encoder = nullptr;
+    /**
+     * The Encoder. May be null, depending on the consumer of this structure.
+     */
+    Encoder *encoder = nullptr;
 
-  /**
-   * The motor being used. By default, this is a dual CIM.
-   */
-  frc::DCMotor motor = frc::DCMotor::CIM(2);
-};
-} //ns wom 
+    /**
+     * The motor being used. By default, this is a dual CIM.
+     */
+    frc::DCMotor motor = frc::DCMotor::CIM(2);
```

Comment:
ngl should probably update this to a neo or something, i doubt we will use a cim on one of our robots in the future

Reply:
epic idea fr fr does frc::DCMotor have those?

Reply:
neo :+1: